website/docs/language/settings/terraform-cloud.html.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: "language"
+page_title: "Terraform Cloud Configuration - Terraform Settings - Configuration Language"
+sidebar_current: "docs-config-terraform"
+description: "The nested `cloud` block configures Terraform's integration with Terraform Cloud."
+---
+
+# Terraform Cloud Configuration
```

Comment:
Should we say something about the advantages of the TFC integration on this page? It seems like one of the main opportunities we'll have to convince people to switch to the cloud integration instead of the `remote` backend, so this would be a good place to mention benefits.